scripts/deployGoerliGovParamUpdate.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  L1AddressRegistry__factory,
+  L1SetInitialGovParamsAction__factory,
+  ArbGoerliSetInitialGovParamsAction__factory,
+  ArbOneGovAddressRegistry__factory,
+} from "../typechain-types";
+import { Wallet, utils } from "ethers";
+import { JsonRpcProvider } from "@ethersproject/providers";
+
+import { ContractVerifier } from "./ContractVerifier";
+import dotenv from "dotenv";
+const abi = utils.defaultAbiCoder;
+
+const GOERLI_ROLLUP_INBOX = "0x6BEbC4925716945D46F0Ec336D5C2564F419682C";
+const GOERLI_ROLLUP_GOV_L1_TIMELOCK = "0x364188EcF8E0733cB90d8EbeD90d56E56205dDfE";
+const GOERLI_ROLLUP_CORE_GOV = "0xa584d185244DCbCa8A98dBdB4e550a5De3A64c81";
+const GOERLI_ROLLUP_TREASURY_GOV = "0x5C7f047cC1564b120DEabFC29B67c60a7c6d6BD2";
+const GOERLI_ROLLUP_TREASURY_WALLET = "0xDA4d3D030B469c3D42B0613202341a6b00E8836e";
+
+dotenv.config();
+
+const l1RPC = "https://goerli.infura.io/v3/8838d00c028a46449be87e666387c71a";
```

Comment:
Maybe we shouldn't upload api keys, how about just loading RPCs from env file? We have existing sample vars for this: https://github.com/ArbitrumFoundation/governance/blob/f80b9c7d100bc7cb45f3182489e3d8f071d79d0a/files/local/.env-sample#L8